src/KokkosPluginLoader.cpp

```diff
@@ -0,0 +1,136 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "RAJA/util/KokkosPluginLoader.hpp"
+
+#include <dlfcn.h>
+#include <dirent.h>
+
+inline
+bool
+isSharedObject(const std::string filename)
+{
+  return (filename.size() > 3 && !filename.compare(filename.size() - 3, 3, ".so"));
+}
+
+namespace RAJA {
+namespace util {
+  
+  KokkosPluginLoader::KokkosPluginLoader()
+  {
+    char *env = ::getenv("KOKKOS_PLUGINS");
+    if (nullptr == env)
```

Comment:
Pedantic, but flip the operands `(env == nullptr)`

Reply:
👍